config/ogdl/src/main/java/org/apache/shiro/config/ogdl/CommonsInterpolator.java

```diff
@@ -18,10 +18,7 @@
  */
 package org.apache.shiro.config.ogdl;
 
-import org.apache.commons.configuration2.interpol.ConfigurationInterpolator;
-import org.apache.commons.configuration2.interpol.ConstantLookup;
-import org.apache.commons.configuration2.interpol.EnvironmentLookup;
-import org.apache.commons.configuration2.interpol.SystemPropertiesLookup;
+import org.apache.commons.configuration2.interpol.*;
```

Comment:
Please don't use star imports. Those are usually considered bad form. Individual imports are great.

Reply:
Thanks for pointing that out! I didn't notice my IDE adding that.